skrub/tests/test_gap_encoder.py

```diff
@@ -196,6 +196,45 @@ def test_get_feature_names_out(n_samples=70):
     return
 
 
+def test_get_feature_names_out_no_words():
+    # Test the GapEncoder get_feature_names_out when there are no words
+    enc = GapEncoder(random_state=42)
+    # A dataframe with words too short
+    df = pd.DataFrame(
+        20
+        * [
+            [
+                "a b c d",
+            ],
+        ],
+    )
+
+    enc.fit(df)
+    # The difficulty here is that, in this specific case short words
+    # should not be filtered out
+    enc.get_feature_names_out()
+    return
+
+
+def test_get_feature_names_out_redundent():
+    # With the following dataframe, the GapEncoder produces feature names
```

Comment:
maybe "used to produce" rather than "produces", with a link to the issue saying that this is a non-regression test?